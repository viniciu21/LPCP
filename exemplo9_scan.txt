declaration:
    a: int;
    b: int;
    c: int;
    x: int;
end_declaration

main:
    a = scan("Oi ");
end_main