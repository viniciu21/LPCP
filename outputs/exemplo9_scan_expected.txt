declaration:
    a: int;
end_declaration

main:
    a = scan("Oi ");
end_main